lms/djangoapps/program_enrollments/models.py

```diff
@@ -0,0 +1,65 @@
+# -*- coding: utf-8 -*-
+"""
+Django model specifications for the Program Enrollments API
+"""
+from __future__ import unicode_literals
+
+from django.contrib.auth.models import User
+from django.db import models
+from django.utils.translation import ugettext as _
+from model_utils.models import TimeStampedModel
+from simple_history.models import HistoricalRecords
+
+
+class ProgramEnrollment(TimeStampedModel):  # pylint: disable=model-missing-unicode
+    """
+    This is a model for Program Enrollments from the registrar service
+
+    .. pii: PII is found in the email address and the external key for a program enrollment
+    .. pii_types: email, external_user_key
```

Comment:
I believe this has to be `email_address, other` - see the `.pii_annotations.yml` file in the root of this repo, it specifies choices for pii_types.

Reply:
Thanks for that one!